### ISSUE TITLE
FATAL FRAME: Maiden of Black Water black screen ingame

### ISSUE THREAD

**Issue by weter11** [doitsujin/dxvk#3065]
Please describe your issue as accurately as possible.

After intro screen, when intro movie start, screen became black, skip it and screen also black, but music is playing. I check logs, but there are no any errors. Also I tried to do apitrace, but game crashes. Also according to reddit talks, for some people disabling AA, DOF and set res to 1280x720 help with black screen, but sadly not in my situation. And this game don't want to cooperate with mangohud, when mangohud enabled in menu and ingame 3.5fps, disabled: 40-45fps.

### Software information
FATAL FRAME / PROJECT ZERO: Maiden of Black Water
Res: 1280x720, AA and DOF disabled

### System information
- GPU: RTX 3070 mobile [GA104M]
- Driver: 515.49.24
- Wine version: Proton experimental 7.0, Tkg proton 7.17, GE Proton 7.33
- DXVK version: 2.0

### Apitrace file(s)
- Put a link here

For instructions on how to use apitrace, see: https://github.com/doitsujin/dxvk/wiki/Using-Apitrace

### Log files
- d3d9.log: [FATAL_FRAME_MOBW_d3d9.log](https://github.com/doitsujin/dxvk/files/10004288/FATAL_FRAME_MOBW_d3d9.log)
- d3d11.log: [FATAL_FRAME_MOBW_d3d11.log](https://github.com/doitsujin/dxvk/files/10004293/FATAL_FRAME_MOBW_d3d11.log)
- dxgi.log: [FATAL_FRAME_MOBW_dxgi.log](https://github.com/doitsujin/dxvk/files/10004311/FATAL_FRAME_MOBW_dxgi.log)

**Comment by weter11** [comment]
Game crash with dlls from apitrace. Is it Windows only? I don't have that OS sadly.

**Comment by Blisto91** [comment]
We can try a quick simple thing to see if it helps.

At the same place your dxvk logs appeared make a file called `dxvk.conf` and insert `dxgi.deferSurfaceCreation = True`
See if that changes anything.

**Comment by weter11** [comment]
Yeah, first I tried all options in dxvk.conf with dxgi... Sadly no any difference.
Also game can start w/o DXVK, but crash after pressing start new game. I think it's because `ID3D11Device::OpenSharedResource` not implemented in WineD3D.

**Comment by Evernow** [comment]
I cannot reproduce this issue on the following setups:

Arc A770 (Linux)
GTX 1630 (Linux) 

I have an apitrace (captured on Windows), for some reason it takes a while to replay during which time there *is* a black screen, but around frame 13000 it starts showing in-game. This is not observed when playing the game. 

https://drive.google.com/file/d/1oq_xiSpMhhg2BNkiqwzkSsd1bbsFlN0I/view?usp=sharing

For anyone testing this game out, do note it's a badly made mess. On the "Press any button" apparently it means any button except enter, left click, right click, or the other 20 keys I smashed. It accepted tab on that screen. K is to click, for some reason. Shift is to exit. And the entire game is single threaded.

**Comment by weter11** [comment]
Looks like where was a problem with manually installed driver from nvidia website. Now I installed a new one from Nvidia PPA and everything ok. Sadly new 525 beta drivers from PPA do not have 32bit packages, so Steam not more working, but I found some workaround how to eliminate that problem. Also my GPU finally working on it's full power and use all 140W TGP instead of 115W.
Sorry for disturb and take your time. 
Best wishes!
